modules/apps/frontend-js/frontend-js-clay-sample-web/.npmbundlerrc

```diff
@@ -0,0 +1,9 @@
+{
+	"config": {
+		"imports": {
+			"@liferay/frontend-js-state-web": {
+				"/": ">=1.0.0"
+			}
+		}
```

Comment:
> My first thought was to add the `useLiferayState` in the `@liferay/frontend-js-react-web` module so that the internal people use it without knowing the state primitives since the hook abstracts a lot of them but I don't have strong opinions for that and anyway I'm fine with any decision.

Yep, that would be the most logical place for it. Only reason I didn't put it there in this PR is that that module isn't using a TS build yet, so I just put all the TS together in `frontend-js-state-web`. But once the TS build is "proven" here, and we've got a workable solution for modules accessing each other's types, it makes sense to move it there. (I'm worried about how hard the "workable solution for modules accessing each other's types" might be, but maybe I'm wrong... 🤞)